src/pages/payment-methods/_payment-data.tsx

```diff
@@ -64,84 +108,78 @@ const StyledRefLink = styled(LocalizedLink)`
-                deposit_time: <Localize translate_text="Instant" />,
-                withdrawal_time: <Localize translate_text="Not applicable" />,
-                description: (
-                    <Localize translate_text="Deposit and withdraw easily in THB, IDR, and VND." />
-                ),
+                min_max_deposit: '_t_10 - 10,000_t_',
+                min_max_withdrawal: '_t_Not applicable_t_',
+                deposit_time: '_t_Instant_t_',
+                withdrawal_time: '_t_Not applicable_t_',
+                description: '_t_Deposit and withdraw easily in THB, IDR, and VND._t_',
                 name: 'DragonPhoenix',
                 reference: 'dragonphoenix-payment-method.pdf',
             },
             {
                 method: <StyledIcon src={ZingPay} alt="zingPay" />,
                 currencies: 'USD EUR AUD GBP',
-                min_max_deposit: <Localize translate_text="10 - 500" />,
-                min_max_withdrawal: <Localize translate_text="10 - 6,000" />,
-                deposit_time: <Localize translate_text="Instant" />,
-                withdrawal_time: <Localize translate_text="1 working day" />,
-                description: (
-                    <Localize translate_text="Payments simplified. Fast and reliable way to deposit funds in your local currency." />
-                ),
+                min_max_deposit: '_t_10 - 500_t_',
+                min_max_withdrawal: '_t_10 - 6,000_t_',
+                deposit_time: '_t_Instant_t_',
+                withdrawal_time: '_t_1 working day_t_',
+                description:
+                    '_t_Payments simplified. Fast and reliable way to deposit funds in your local currency._t_',
                 name: 'ZingPay',
                 reference: 'zingpay-payment-method.pdf',
             },
             {
                 method: <StyledIcon src={NganLuong} alt="NganLuong" />,
                 currencies: 'USD',
-                min_max_deposit: <Localize translate_text="10 - 2,500" />,
-                min_max_withdrawal: <Localize translate_text="10 - 2,500" />,
-                deposit_time: <Localize translate_text="1 working day" />,
-                withdrawal_time: <Localize translate_text="1 - 3 working days" />,
-                description: (
-                    <Localize translate_text="A leading e-wallet service in Vietnam that’s easy to use, flexible, and safe. Used by thousands of merchants." />
-                ),
+                min_max_deposit: '_t_10 - 2,500_t_',
+                min_max_withdrawal: '_t_10 - 2,500_t_',
+                deposit_time: '_t_1 working day_t_',
+                withdrawal_time: '_t_1 - 3 working days_t_',
+                description:
+                    '_t_A leading e-wallet service in Vietnam that’s easy to use, flexible, and safe. Used by thousands of merchants._t_',
                 name: 'NganLuong',
                 reference: 'nganluong-payment-method.pdf',
                 locales: ['vi'],
```

Comment:
Can you check are we still using this property?
If no, let's remove it and type for that